### PR TITLE
Enable opensource images multi-platform building

### PR DESCRIPTION
**Pull request by MasonXon**

### CODE REVIEW

**Comment by MasonXon** [comment]
@ganisback

**Comment by Rader** [comment]
@ganisback @SeanHH86 please help review

File: docker/finetune/build.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# 检查参数是否传入
```
Comment (ganisback) [inline docker/finetune/build.sh:3]: please use english

File: docker/inference/build.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# 检查参数是否传入
```
Comment (ganisback) [inline docker/inference/build.sh:3]: same to above

**Comment by ganisback** [comment]
@MasonXon the fix looks good, just a little comment

File: docker/finetune/build.sh (hunk 1 above)
Comment (MasonXon) [reply docker/finetune/build.sh]: Already removed

**Review by ganisback — APPROVED** [review]

**Pull request merged**